CosNet.API/Data/DbContexts/DbSeeder.cs

```diff
@@ -67,6 +67,93 @@ public static void Seed(ApplicationDbContext context)
                 };
                 context.Cosplays.AddRange(cosplays);
                 context.SaveChanges();
+
+                var cosplayItems = new List<CosplayItem>
```

Comment:
We should be carefull keeping this file organized, if we end up with 20 database tables we dont want a 5000 line seeder file